lib/internal/fs/promises.js

```diff
@@ -276,12 +275,9 @@ class FileHandle extends EventEmitter {
   /**
    * @typedef {import('../webstreams/readablestream').ReadableStream
    * } ReadableStream
-   * @param {{
-   *   type?: string;
-   *   }} [options]
    * @returns {ReadableStream}
    */
-  readableWebStream(options = kEmptyObject) {
```

Comment:
While this API is still considered to be experimental it has been around for a bit and we need to be careful with changes like this. I would recommend keeping the `options` argument for now and emitting a warning if the `options` is not `undefined` and the `options.type` is not `bytes` so that it is at least more obvious that there is a notable change in behavior on this.

Reply:
Okay. This is my first attempt at making a nontrivial contribution to nodejs. Can you point me to any examples I can ape?

Reply:
do a search for `process.emitWarning(...)` for some clues :-)

Reply:
Problem: there is no specified non-'bytes' `type` for ReadableStream.  Nobody would have any cause to pass such a parameter; `undefined` is the only thing with the proper semantics.

https://streams.spec.whatwg.org/#underlying-source-api
https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/ReadableStream#type

Reply:
@jasnell ping, I don’t think there’s anything we can do here in terms of warnings.

Reply:
@jasnell ping 🌞.

Reply:
@jasnell ping. If you can't review this any longer, can you suggest someone who can?

Reply:
> Problem: there is no specified non-`'bytes'` `type` for ReadableStream.  Nobody would have any cause to pass such a parameter; `undefined` is the only thing with the proper semantics.

So, if `bytes` isn't set, it'll be done so under-the-hood, or there's a `bytes` type and an `undefined` type?

Reply:
Thanks for the review.

[There's a `'bytes'` type and an `undefined` type]( https://streams.spec.whatwg.org/#dom-underlyingsource-type), yes:
> Setting any value other than "[bytes](https://streams.spec.whatwg.org/#dom-readablestreamtype-bytes)" or undefined will cause the [ReadableStream()](https://streams.spec.whatwg.org/#rs-constructor) constructor to throw an exception.

The goal is to make it a bytes stream no matter what.  I will proceed without a warning.

Reply:
I don't follow what you mean by `options.byteStream` or `true`.  Neither of those things are in the existing API.

More generally, a byte-oriented ReadbleStream is supposed to be, from the caller's view, a superset of the functionality of a normal one.  A byte-oriented stream should behave the same as a normal one if you don't pass `mode: 'byob'` to `getReader`, which is only an option on byte-oriented streams in the first place.  Not returning a byte-oriented stream when your stream exclusively vends bytes (as is the case here) needlessly constrains what callers can do.

Reply:
Sorry! I'm quite tired, and I misworded (extremely) my idea. I'll open a review comment with my suggested change, and this time i'll word my message correctly.